prettier.config.js

```diff
@@ -2,4 +2,6 @@ module.exports = {
   arrowParens: "always",
   endOfLine: "lf",
   trailingComma: "all",
+
+  plugins: ["prettier-plugin-elm", "prettier-plugin-packagejson"],
```

Comment:
Plugin search feature [has been removed in Prettier v3](https://prettier.io/blog/2023/07/05/3.0.0.html#plugin-search-feature-has-been-removed-14759httpsgithubcomprettierprettierpull14759-by-fiskerhttpsgithubcomfisker), so listing Prettier plugins manually.